[2025-05-02T02:33:52.008+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Pipeline-proyecto2.load_raw_data manual__2025-05-02T02:33:48.627202+00:00 [queued]>
[2025-05-02T02:33:52.024+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Pipeline-proyecto2.load_raw_data manual__2025-05-02T02:33:48.627202+00:00 [queued]>
[2025-05-02T02:33:52.025+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-05-02T02:33:52.043+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): load_raw_data> on 2025-05-02 02:33:48.627202+00:00
[2025-05-02T02:33:52.056+0000] {standard_task_runner.py:57} INFO - Started process 77 to run task
[2025-05-02T02:33:52.064+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Pipeline-proyecto2', 'load_raw_data', 'manual__2025-05-02T02:33:48.627202+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/dag-proyecto2.py', '--cfg-path', '/tmp/tmpu2vu15cu']
[2025-05-02T02:33:52.067+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask load_raw_data
[2025-05-02T02:33:52.142+0000] {task_command.py:410} INFO - Running <TaskInstance: Pipeline-proyecto2.load_raw_data manual__2025-05-02T02:33:48.627202+00:00 [running]> on host 5b68dcb6aec6
[2025-05-02T02:33:52.261+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Pipeline-proyecto2' AIRFLOW_CTX_TASK_ID='load_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-02T02:33:48.627202+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-02T02:33:48.627202+00:00'
[2025-05-02T02:34:13.977+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-02T02:34:13.995+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=Pipeline-proyecto2, task_id=load_raw_data, execution_date=20250502T023348, start_date=20250502T023352, end_date=20250502T023413
[2025-05-02T02:34:14.112+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-02T02:34:14.160+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
